src/scripts/creature-stats.js

```diff
@@ -545,9 +545,8 @@ function getCustomActionsData() {
 
         if (rolls) {
             for (let dmg of rolls) {
-                if (dmg.damage_type) {
-                    element.insertAdjacentHTML('beforeend', `<button class="btn--attack btn--hover">${dmg.damage_dice} ${dmg.damage_type.index}</button>`);
-                }
+                damage = separateDmgRoll(dmg);
```

Comment:
Any thoughts about using [object deconstruction here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)?
```
const { damageDice, damageType }  = separateDmgRoll(dmg);
```

Reply:
omg I thought I was going crazy earlier, guess I forgot to add const lol.